postman-generator/src/main/scala/models/attributes/PostmanAttributes.scala

```diff
@@ -2,9 +2,10 @@ package models.attributes
 
 import play.api.libs.json.Json
 
-object PostmanBasicAuthAttribute {
+object PostmanAttributes {
```

Comment:
there is also `ObjectReferenceAttribute` - if we want to merge stuff, it would be nice if it was here as well

Reply:
done